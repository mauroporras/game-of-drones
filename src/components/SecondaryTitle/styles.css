.SecondaryTitle {
  color: var(--color-dark-gray);
  margin: 0 0 2rem;
  text-align: center;
}
