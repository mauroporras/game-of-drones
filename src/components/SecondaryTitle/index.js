import React from 'react'

import './styles.css'

const SecondaryTitle = props => (
  <h2 className='SecondaryTitle'>{props.text}</h2>
)

export default SecondaryTitle
