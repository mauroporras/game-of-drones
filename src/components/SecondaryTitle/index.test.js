import React from 'react'
import renderer from 'react-test-renderer'

import SecondaryTitle from './index'

test("Component renders the correct text", () => {
  const component = renderer.create(
    <SecondaryTitle text='Hello' />
  )
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
