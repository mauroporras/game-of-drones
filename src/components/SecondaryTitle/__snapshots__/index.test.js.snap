exports[`test Component renders the correct text 1`] = `
<h2
  className="SecondaryTitle">
  Hello
</h2>
`;
